### PR TITLE
Speaker updates after #42

### PR DESCRIPTION
**Pull request by cmil**
This regenerates the TEI files with new speaker JSONs from pull request #42.

### CODE REVIEW

**Comment by cmil** [comment]
@lucagiovannini7 Here are the regenerated TEI files. Please double check that these are the changes you intended to make.

**Review by lucagiovannini7 — APPROVED** [review]
Everything seems in order to me. I left a comment, but the issue is not so relevant, the text could remain as it is for the time being. Thanks for your time!

**Comment by lucagiovannini7** [comment]
Could you please approve these changes? I'm currently working on these plays and I need the updated files. Thanks!

File: tei/dekker-the-noble-spanish-soldier.xml [hunk 1]
```diff
@@ -347,61 +338,61 @@
               <body>
                 <div type="song" xml:id="ep000235-e101120">
                   <head>Song, </head>
-                  <sp who="#ep000235-question" xml:id="ep000235-e101130">
+                  <sp who="#ep000235-" xml:id="ep000235-e101130">
                     <speaker>Quest. </speaker>
```
[the remark this answers is not in the document]
Comment (cmil) [reply tei/dekker-the-noble-spanish-soldier.xml:342]: For now I just removed the empty IDs and `who` attributes with invalid references.

**Comment by lucagiovannini7** [comment]
@cmil the last update of EPDraCor on the staging.dracor.org server was on the 19.11.2022, so the modification from this merge are not live yet. Would you mind refreshing the corpus, so that I can grab the updated XMLs (I know I could download the files manually, but they're quite a few and I'm working with the API)? This is the last you'll hear from me for today and the next weeks, I promise.

**Pull request merged**